src/utils/index.ts

```diff
@@ -17,7 +17,7 @@ export const isEmptyString = <T,>(value: string | T): boolean =>
  *
  */
 export const isValidNumber = <T,>(value: number | string | T): boolean =>
-    !isNaN(Number(value));
+    typeof value === 'number';
```

Comment:
@KelseyJr Can you please explain what was the issue in the previous implementation?

Reply:
When you convert a empty array/string to number, Javascript will return 0. So I made this changes, but this implementation is wrong, due the comments below. I'll fix it.